lib/store.js

```diff
@@ -224,7 +241,7 @@ internals.coerce = function (value, type, options) {
             break;
         case 'array':
             if (typeof value === 'string') {
-                result = value.split(options.splitToken);
+                result = value ? value.split(options.splitToken) : [];
```

Comment:
Nice catch on that one, completely forgot this edge case, when splitting an empty string it'll return an array of a single empty string....

Reply:
I only noticed it because this case previously wasn't hit by the test suite.  When I removed the process.env-patching code in the test suite, this popped up with a test failure :)